docs/proposals/federation.md

```diff
@@ -156,6 +164,7 @@ It seems that most of this boils down to:
 1. **cross-cluster migration** (how do compute and storage resources,
    and the distributed applications to which they belong, move from
    one cluster to another)
+1. cross-cluster monitoring and auditing (a.k.a. Unified visibility)
```

Comment:
Multi-cluster load balancing is discussed under H.A. elsewhere, but you're right, it belongs here.  I'll move stuff around to make it more sensible.